EasyPost/Webhook.cs

```diff
@@ -98,5 +100,29 @@ public static async Task<Webhook> Retrieve(string id)
 
             return await request.Execute<Webhook>();
         }
+
+        public static Event ValidateWebhook(byte[] data, Dictionary<string, object?> headers, string secret)
+        {
+            string? providedSignature = string.Empty;
+            if (headers.ContainsKey("X-Hmac-Signature"))
+            {
+                providedSignature = headers["X-Hmac-Signature"]?.ToString();
+            }
+            else
+            {
+                throw new Exception("Webhook received does not contain an HMAC signature.");
+            }
+
+            string computedHexDigest = data.CalculateHMACSHA256HexDigest(secret, NormalizationForm.FormKD); // normalize with NFKD profile
```

Comment:
Issue: Comments that say what a thing does when the thing has the same info as the comment don't provide value and shouldn't be included as they only add noise to the code.

```suggestion
            string computedHexDigest = data.CalculateHMACSHA256HexDigest(secret, NormalizationForm.FormKD);
```